src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -712,6 +712,9 @@ export default class ConvasController {
         if (aircraft.category === FLIGHT_CATEGORY.DEPARTURE) {
             return;
         }
```

Comment:
There should be an empty line between if statements here. 

Actually, I think this would read better if there were a method that n the `GameController` that simply returned a Boolean. That way we could combine the ifs and use an `||` within a single if block with a small single return.